metadash/src/store.js

```diff
@@ -120,7 +120,15 @@ const store = new Vuex.Store({
         msg: text,
         alert: true
       }
-      state.alerts.push(alert)
+      if (noRepeat) {
```

Comment:
Should this be noDuplicate or am I missing anything?

Reply:
Fixed it. Sorry for being so careless!